.github/workflows/codeQL-scanning.yaml

```diff
@@ -0,0 +1,49 @@
+name: "CodeQL scanning"
+
+on:
+  push:
+    branches:
+      - master
+      - 'release/*'
+  pull_request:
+    branches:
+      - main
```

Comment:
There is no `main` branch

Reply:
we should probably change to main. `master` seems deprecated as the default name.